app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -178,6 +181,19 @@ def hash_for_pdf
           Text36: mfs_spouse_ssn[8],
         })
       end
+      # line 65 nj child tax credit
+      # TODO: what should we do if there are more digits to fill?
```

Comment:
I vaguely recollect something about attaching a piece of paper. What should we do in this field if the value is greater than $9999?

Reply:
I think this needs to be a PM question

Reply:
good question, I'll raise in the CFA channel